src/dynamics/ads_player/ads_player.js

```diff
@@ -331,22 +338,33 @@ Scoped.define("module:Ads.Dynamics.Player", [
 
                 create: function() {
                     const dynamics = this.parent();
-                    const isMobile = Info.isMobile();
                     const adContainer = this.getAdContainer();
+                    const IMASettings = this.get("imasettings");
                     const adManagerOptions = {
                         adContainer: adContainer,
                         adsRenderingSettings: this.get("imaadsrenderingsetting"),
-                        IMASettings: this.get("imasettings")
-                    };
-
-                    if (isMobile && Info.iOSversion().major >= 10) {
-                        adManagerOptions.IMASettings.iOS10Plus = true;
+                        videoElement: this.getVideoElement() || null,
```

Comment:
IMA SDK suggest we always pass in a video element:
![Screenshot 2024-07-30 at 2 51 06 PM](https://github.com/user-attachments/assets/75978ebc-0422-4ae3-baae-b8b31c34fac6)